Blueprints/BlueprintDefinitions/vs2022/AnnotationsFramework/template/test/BlueprintBaseName.1.Tests/FunctionsTest.cs

```diff
@@ -1,20 +1,32 @@
 using Xunit;
 using Amazon.Lambda.TestUtilities;
+using Moq;
 
-namespace BlueprintBaseName._1.Tests;
 
-public class FunctionsTest
+namespace BlueprintBaseName._1.Tests
```

Comment:
nit: could we stick to using top-level statements?

Reply:
Done